src/vma/dev/qp_mgr.cpp

```diff
@@ -123,10 +124,53 @@ qp_mgr::~qp_mgr()
 	munmap(m_rq_wqe_idx_to_wrid, m_rx_num_wr * sizeof(*m_rq_wqe_idx_to_wrid));
 	munmap(m_sq_wqe_idx_to_wrid, m_tx_num_wr * sizeof(*m_sq_wqe_idx_to_wrid));
 #endif // DEFINED_VMAPOLL
+
+#ifndef DEFINED_VMAPOLL
+	delete[] m_rq_wqe_idx_to_wrid;
+#else
+	free(m_rq_wqe_idx_to_wrid);
+#endif
 	qp_logdbg("Rx buffer poll: %d free global buffers available", g_buffer_pool_rx->get_free_count());
 	qp_logdbg("delete done");
 }
 
+static inline uint32_t align32pow2(uint32_t x)
```

Comment:
This must go in a util file, not here. Maybe in _src/vma/util/utils.h_?
It should also have a comment explaining what it does.